tutorial_guide/arc_server_onboard_certificates/_index.md

```diff
@@ -0,0 +1,6 @@
+# Overview
+Azure Arc version 1.41 introduces certificate-based authentication for connecting and disconnecting servers, replacing the old method of using passwords. This new feature makes managing servers easier and more secure.
+
```

Comment:
Can we include _index.md and guide.md into the same _index.md file?

Reply:
I have moved the content from guide.md to _index.md

Reply:
Thanks